bin/sqlitedb.py

```diff
@@ -108,7 +110,7 @@ def close_json(self, file_name):
     if not re.search(r'\.json$', file_name):
       raise Exception('wrong output file type: "' + file_name + '"' )
     with open(file_name, mode='a') as fd:
-      fd.write(']}\n');
+      fd.write('}')
```

Comment:
We might need '\n' there?

Reply:
json does not need it. It loads in chrome tracing with or without it